diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$( "li" )
+$('#OL>li')
```

Comment:
This will select `<li>` elements that are children of an element that has an `id` of `ol`.  `$('ol > li')` is probably what you meant.